smack-repl/src/main/java/org/igniterealtime/smack/smackrepl/WebsocketConnection.java

```diff
@@ -0,0 +1,49 @@
+/**
+ *
+ * Copyright 2019 Aditya Borikar
+ *
+ * This file is part of smack-repl.
+ *
+ * smack-repl is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+ */
+package org.igniterealtime.smack.smackrepl;
+
+import java.io.IOException;
+
+import org.jivesoftware.smack.SmackException;
+import org.jivesoftware.smack.XMPPException;
+
+import org.jivesoftware.smack.c2s.ModularXmppClientToServerConnection;
+import org.jivesoftware.smack.c2s.ModularXmppClientToServerConnectionConfiguration;
+import org.jivesoftware.smack.tcp.XmppTcpTransportModuleDescriptor;
+
+public class WebsocketConnection {
+
+    public static void main(String[] args) throws SmackException, IOException, XMPPException, InterruptedException {
+
+        ModularXmppClientToServerConnectionConfiguration.Builder builder = ModularXmppClientToServerConnectionConfiguration.builder();
+
+        builder.removeModule(XmppTcpTransportModuleDescriptor.class);
```

Comment:
That approach is fragile. If we ever add another module later, then this will be also active here. What you want to do instead is `removeAllModules()` and then add the one module you want.